### PR TITLE
docs: add prometheus blog

### PR DESCRIPTION
**Pull request by guitu168**
Fixes: #[Add issue number here]

Changes:

Screenshots of the change:

### CODE REVIEW

File: website/i18n/zh/docusaurus-plugin-content-blog/2022/07/13/monitor-api-gateway-apisix-with-prometheus.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+
+## 背景信息
+
+Apache APISIX 是一个动态、实时、高性能的 API 网关，提供了负载均衡、动态上游、灰度发布、服务熔断、身份认证、可观测性等丰富的流量管理功能。作为 API 网关，Apache APISIX 不仅拥有丰富的插件，而且支持插件的热加载。
+
+Prometheus 是一款开源的云原生监控报警系统，它为用户提供强大的指标存储功能、告警功能和近实时查询引擎，支持多种 exporter 采集数据和 pushgateway 进行数据上报。Prometheus 的性能足够支撑上万台规模的集群，是目前最流行的开源监控系统之一。
+
+## 插件介绍
+
+APISIX 通过 Prometheus Exporter 以 HTTP 的方式向外暴露 Apache APISIX 的设定的监控指标，包括 Apache APISIX 的连接数、HTTP 请求的状态码分布、HTTP 请求总数、HTTP 请求延时等重要指标。你可以使用 `prometheus` 插件，来监控 APISIX 的各个指标，及时地了解系统运行状态。
```
Comment (SylviaBABY) [inline website/i18n/zh/docusaurus-plugin-content-blog/2022/07/13/monitor-api-gateway-apisix-with-prometheus.md]:
```suggestion
APISIX 通过 Prometheus Exporter 以 HTTP 的方式向外暴露 Apache APISIX 设定的监控指标，包括 Apache APISIX 的连接数、HTTP 请求的状态码分布、HTTP 请求总数、HTTP 请求延时等重要指标。你可以使用 `prometheus` 插件，来监控 APISIX 的各个指标，及时地了解系统运行状态。
```

File: website/i18n/zh/docusaurus-plugin-content-blog/2022/07/13/monitor-api-gateway-apisix-with-prometheus.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+
+## 配置步骤
+
+### 步骤一：启用 prometheus 插件
+
+首先，你需要安装完成 APISIX，本文所有操作均在 CentOS 7.6 中，请在进行该操作前，参考 [APISIX 安装指南](https://apisix.apache.org/zh/docs/apisix/how-to-build/)完成 APISX 的安装。
```
Comment (SylviaBABY) [inline website/i18n/zh/docusaurus-plugin-content-blog/2022/07/13/monitor-api-gateway-apisix-with-prometheus.md]:
```suggestion
首先，你需要安装完成 APISIX，本文所有操作均在 CentOS 7.6 中进行，请在执行该操作前，参考 [APISIX 安装指南](https://apisix.apache.org/zh/docs/apisix/how-to-build/)完成 APISX 的安装。
```

File: website/i18n/zh/docusaurus-plugin-content-blog/2022/07/13/monitor-api-gateway-apisix-with-prometheus.md [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+
+APISIX 默认启用 `prometheus` 插件，该插件会增加 `/apisix/prometheus/metrics` 接口，并且默认会通过独立的服务地址暴露指标，默认服务地址是 `127.0.0.1:9091`，本文将使用默认服务地址。
```
Comment (SylviaBABY) [inline website/i18n/zh/docusaurus-plugin-content-blog/2022/07/13/monitor-api-gateway-apisix-with-prometheus.md]:
```suggestion
APISIX 默认启用 `prometheus` 插件，该插件会增加 `/apisix/prometheus/metrics` 接口，并且默认会通过独立的服务地址暴露指标（默认服务地址是 `127.0.0.1:9091`，本文将使用默认服务地址）。
```

File: website/i18n/zh/docusaurus-plugin-content-blog/2022/07/13/monitor-api-gateway-apisix-with-prometheus.md [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+
+```yaml
+plugin_attr:
+  prometheus:
+    export_addr:
+      ip: 127.0.0.1
+      port: 9092
+```
+
+建议在 `ip` 参数中配置内网地址，确保仅在局域网内暴露 Prometheus Exporter 接口，当然你也可以使用 [public-api](https://apisix.apache.org/zh/docs/apisix/plugins/public-api/) 插件保护此接口。为了方便在测试环境中进行测试，你也可以将 Prometheus Exporter 直接暴露在数据面的端口上。如下所示：
```
Comment (SylviaBABY) [inline website/i18n/zh/docusaurus-plugin-content-blog/2022/07/13/monitor-api-gateway-apisix-with-prometheus.md]:
```suggestion
建议在 `ip` 参数中配置内网地址，确保仅在局域网内暴露 Prometheus Exporter 接口，当然你也可以使用 [public-api](https://apisix.apache.org/zh/docs/apisix/plugins/public-api/) 插件保护此接口。为了方便在测试环境中进行，你也可以将 Prometheus Exporter 直接暴露在数据面的端口上。如下所示：
```

File: website/i18n/zh/docusaurus-plugin-content-blog/2022/07/13/monitor-api-gateway-apisix-with-prometheus.md [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+# TYPE apisix_etcd_modify_indexes gauge
+apisix_etcd_modify_indexes{key="consumers"} 0
+...
+# TYPE apisix_node_info gauge
+apisix_node_info{hostname="APISIX"} 1
+```
+
+### 步骤二：配置 Prometheus
+
+1. 通过安装 Prometheus：
```
Comment (SylviaBABY) [inline website/i18n/zh/docusaurus-plugin-content-blog/2022/07/13/monitor-api-gateway-apisix-with-prometheus.md]:
```suggestion
1. 安装 Prometheus：
```

File: website/i18n/zh/docusaurus-plugin-content-blog/2022/07/13/monitor-api-gateway-apisix-with-prometheus.md [hunk 6]
```diff
@@ -0,0 +1,219 @@
...
+
+```shell
+cd /usr/local && \
+wget https://github.com/prometheus/prometheus/releases/download/v2.33.0-rc.0/prometheus-2.33.0-rc.0.linux-amd64.tar.gz && \
+tar -zxvf prometheus-2.33.0-rc.0.linux-amd64.tar.gz && \
+cd /usr/local/prometheus-2.33.0-rc.0.linux-amd64/
+```
+
+2. APISIX暴露出来的指标，将在 Prometheus 服务端新增一个 Scrape 从而完成指标收集。通过在  Prometheus 的 `prometheus.yml` 配置文件中添加此 URI 地址，就可以自动完成指标数据的提取。可参考以下内容修改配置：
```
Comment (SylviaBABY) [inline website/i18n/zh/docusaurus-plugin-content-blog/2022/07/13/monitor-api-gateway-apisix-with-prometheus.md]:
```suggestion
2. APISIX 暴露出来的指标，将在 Prometheus 服务端新增一个 Scrape 从而完成指标收集。通过在 Prometheus 的 `prometheus.yml` 配置文件中添加此 URI 地址，就可以自动完成指标数据的提取。可参考以下内容修改配置：
```

**Comment by yzeng25** [comment]
You would need an English markdown file to have the preview ready.

File: website/i18n/zh/docusaurus-plugin-content-blog/2022/07/13/monitor-api-gateway-apisix-with-prometheus.md [hunk 7]
```diff
@@ -0,0 +1,211 @@
...
+
+## 背景信息
+
+Apache APISIX 是一个动态、实时、高性能的 API 网关，提供了负载均衡、动态上游、灰度发布、服务熔断、身份认证、可观测性等丰富的流量管理功能。作为 API 网关，Apache APISIX 不仅拥有丰富的插件，而且支持插件的热加载。
+
+Prometheus 是一款开源的云原生监控报警系统，它为用户提供强大的指标存储功能、告警功能和近实时查询引擎，支持多种 exporter 采集数据和 pushgateway 进行数据上报。Prometheus 的性能足够支撑上万台规模的集群，是目前最流行的开源监控系统之一。
+
+## 插件介绍
+
+APISIX 通过 Prometheus Exporter 以 HTTP 的方式向外暴露 APISIX 设定的监控指标，包括 APISIX 的连接数、HTTP 请求的状态码分布、HTTP 请求总数、HTTP 请求延时等重要指标。你可以使用 `prometheus` 插件，来监控 APISIX 的各个指标，及时地了解系统运行状态。
```
Comment (SylviaBABY) [inline website/i18n/zh/docusaurus-plugin-content-blog/2022/07/13/monitor-api-gateway-apisix-with-prometheus.md]:
```suggestion
APISIX 通过 Prometheus Exporter 以 HTTP 的方式向外暴露 APISIX 设定的监控指标，包括 APISIX 的连接数、HTTP 请求的状态码分布、HTTP 请求总数、HTTP 请求延时等重要指标。你可以使用 `prometheus` 插件，来监控 APISIX 的各个指标，以便及时地了解系统运行状态。
```

File: website/i18n/zh/docusaurus-plugin-content-blog/2022/07/13/monitor-api-gateway-apisix-with-prometheus.md [hunk 8]
```diff
@@ -0,0 +1,211 @@
...
+1. 安装 Prometheus：
+
+```shell
+cd /usr/local && \
+wget https://github.com/prometheus/prometheus/releases/download/v2.33.0-rc.0/prometheus-2.33.0-rc.0.linux-amd64.tar.gz && \
+tar -zxvf prometheus-2.33.0-rc.0.linux-amd64.tar.gz && \
+cd /usr/local/prometheus-2.33.0-rc.0.linux-amd64/
+```
+
+2. 通过在 Prometheus 的 `prometheus.yml` 配置文件中添加相关配置，Prometheus 就可以自动抓取指标数据。具体配置如下：
```
Comment (SylviaBABY) [inline website/i18n/zh/docusaurus-plugin-content-blog/2022/07/13/monitor-api-gateway-apisix-with-prometheus.md:92]:
```suggestion
2. 通过在 Prometheus 的 `prometheus.yml` 配置文件中添加相关细节，Prometheus 就可以自动抓取指标数据。具体配置如下：
```

File: website/i18n/zh/docusaurus-plugin-content-blog/2022/07/13/monitor-api-gateway-apisix-with-prometheus.md [hunk 9]
```diff
@@ -0,0 +1,211 @@
...
+curl http://127.0.0.1:9080/index.html
+```
+
+返回 Grafana 页面，单击左侧导航栏搜索按钮，就可以查看 APISIX 相关的指标了。
+
+![network error/apisix grafana.png](https://static.apiseven.com/2022/blog/0713/4.png)
+
+### 步骤六：嵌入 APISIX Dashboard
+
+1. 安装并登录 APISIX Dashboard。默认地址是`127.0.0.1:9000`。请参考 [APISIX Dashboard 安装指南](https://apisix.apache.org/zh/docs/dashboard/install/)。
```
Comment (SylviaBABY) [inline website/i18n/zh/docusaurus-plugin-content-blog/2022/07/13/monitor-api-gateway-apisix-with-prometheus.md]:
```suggestion
1. 安装并登录 APISIX Dashboard，默认地址为`127.0.0.1:9000`。具体细节可参考 [APISIX Dashboard 安装指南](https://apisix.apache.org/zh/docs/dashboard/install/)。
```

**Comment by yzeng25** [comment]
what is this pr for? I see it is already published online in https://apisix.apache.org/zh/blog/2022/07/13/monitor-api-gateway-apisix-with-prometheus

**Comment by SkyeYoung** [comment]
@yzeng25 This was my mistake. I included this article in my testing and submitted it directly to the repository in this PR #1218 😭

**Pull request closed without merging**